tests/tests_unit/test_api/test_simulators/conftest.py

```diff
@@ -0,0 +1,20 @@
+import json
+import re
+from collections.abc import Callable
+
+from requests import PreparedRequest
+from responses import RequestsMock
+
+from tests.utils import jsgz_load
+
+
+def add_mocked_request(rsps: RequestsMock, url: str, request_callback: Callable[[dict], tuple[int, dict]]) -> None:
```

Comment:
Using the callback makes this logic a bit hard to follow to be honest. I wonder if we should remove this function entirely?

I use the `callback` when the return value is dynamic, it doesn't seem to be that here? As far as I can see you return:

```py
def request_callback(_: dict) -> tuple[int, dict]:
    response_item = {
        **TEST_ROUTINE_REV_ITEM_RESPONSE_FIELDS,
        **mock_response_fields,
    }
    return (200, response_item)
```

This should allow you to just do this directly:
```py
responses.add(
    "POST",
    url=...,
    json=PAYLOAD,
    status=STATUS,
)
```

Reply:
done